modules/admin/con-rbac-config-permission-policies-external-file.adoc

```diff
@@ -131,3 +131,83 @@ permission:
 ----
 --
 
+
+== Mounting `policy.csv` file using the {product-short} Operator
```

Comment:
This new content is clearly a new procedure. It should go in a new `proc-mounting-the-policy-csv-file-using-the-rhdhoprator.adoc` file.

Reply:
Done - see [`9c11aea` (#499)](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/499/commits/9c11aea9baac31186f8799ddfbfa501b2c2260db).
For consistency, I've done the same thing by moving the Helm chart procedure to a new `proc-mounting-the-policy-csv-file-using-helm.adoc` file - see [`3414e98` (#499)](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/499/commits/3414e98a013d63270e4df8dd39a7e7b7fe0f8443).